test/e2e/specs/published-content/likes__post.ts

```diff
@@ -61,7 +61,7 @@ describe( DataHelper.createSuiteTitle( 'Likes (Post)' ), function () {
 		} );
 	} );
 
-	describe( 'As the liking user', () => {
+	describe( 'As the liking user', function () {
```

Comment:
Actually, now that I approved it, does the change with the bound `this` context and scope make any difference in jest? I guess it should be ok if the tests pass.

Reply:
It doesn't impact the scope - Jest supports both `() =>` and `function()`, but I tend to prefer the `function` syntax.